subsys/nrf_security/src/drivers/cracen/cracenpsa/include/cracen_psa_wpa3_sae.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef CRACEN_PSA_WPA3_SAE_H
+#define CRACEN_PSA_WPA3_SAE_H
+
+#include <stdint.h>
+#include "cracen_psa_primitives.h"
+
+/**
+ * See "PSA Cryptography API" for documentation.
+ */
```

Comment:
Not correct. Please follow Doxygen guidelines: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/dev_model_and_contributions/documentation/styleguide.html#doxygen_guidelines